src/components/textInput/textInput.tsx

```diff
@@ -62,6 +72,81 @@ export default function TextInput(props: TextInputProps) {
     setMessageText(e.target.value)
   }
 
+  function handleOnFocusToggle() {
+    setIsFocused(!isFocused)
+  }
+
+  function handleToggleSpeechToText() {
+    const microphone = new window.webkitSpeechRecognition()
+
+    microphone.lang = 'en-US'
```

Comment:
Do we want to add a props for language setting?

Reply:
I'm realizing I think I can actually omit this as it can default to the HTML `lang` attribute value or the user agent's language setting. Can you think of any use cases where someone might want to explicitly set `lang`?

Ref: https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition/lang

Reply:
I have another idea about this. Instead of having a props for this, maybe we could use `navigator.languages` to get system languages and assign the first one to `microphone.lang`?

Reply:
We could do that. We can use `navigator.language` since it's the first item in `navigator.languages`.

Reply:
Yes, `navigator.language` is better